www/src/components/Search/Search.scss

```diff
@@ -53,7 +55,7 @@
     margin: 6.875rem auto auto;
     cursor: auto;
 
-    @media (max-width: map-get($grid-breakpoints, "md")) {
+    @media (max-width: map.get($grid-breakpoints, "md")) {
```

Comment:
[curious] I noticed some `@media` queries were modified to replace things like `max-width` with `width >=` (e.g., `@media only screen and (width <= 768px) {` in `Toast`).

1. Should we be consistent about using `max-width` and `min-width` vs. just `width` in our media queries?
2. Should any hardcoded pixel values in media queries (like the `Toast` example above) be using the `$grid-breakpoints` via `map.get`?

Reply:
> The width <= 768px seems simple and reasonable to me. That change isn't necessary as far as I know though, so let me know if you see any issues.

No issues immediately observed; I agree the change should be functionally equivalent. The newer syntax is easier definitely easier to grok.

> Probably, but let's do that in a follow-up PR as this one is already huge

Totally, just wanted to call it out for now :)